packaging/docker/env.example

```diff
@@ -0,0 +1,75 @@
+# Copy this file to .env, then update it with your own settings
+
+### CLEF
+
+## chain id to use for signing (1=mainnet, 3=ropsten, 4=rinkeby, 5=goerli) (default: 12345)
+# CLEF_CHAINID=12345
+
+### BEE
+
+## HTTP API listen address (default ":1633")
+# BEE_API_ADDR=:1633
+## initial nodes to connect to (default [/dnsaddr/bootnode.ethswarm.org])
+# BEE_BOOTNODE=[/dnsaddr/bootnode.ethswarm.org]
+## enable clef signer
+# BEE_CLEF_SIGNER_ENABLE=false
```

Comment:
@Eknir should I enable clef and set clef endpoint or should I leave it to the user?

Reply:
Please make it in such a way as to subtly nudge the user to use Clef wherever possible and feasible.

Reply:
@Eknir check README